docs/source/conf.py

```diff
@@ -0,0 +1,46 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
Is this file generated by hand or autogenerated from the README/pyproject.toml?

Reply:
Should be hand-crafted afaik